chromium_src/chrome/browser/ui/views/tabs/tab_slot_controller.h

```diff
@@ -6,10 +6,11 @@
 #ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_TABS_TAB_SLOT_CONTROLLER_H_
 #define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_TABS_TAB_SLOT_CONTROLLER_H_
 
-#define IsGroupCollapsed                                   \
-  IsTabTiled(const Tab* tab) const = 0;                    \
-  virtual bool IsFirstTabInTile(const Tab* tab) const = 0; \
-  virtual const Browser* GetBrowser() const = 0;           \
+#define IsGroupCollapsed                                                      \
+  IsTabTiled(const Tab* tab) const = 0;                                       \
+  virtual bool IsFirstTabInTile(const Tab* tab) const = 0;                    \
+  virtual const Browser* GetBrowser() const = 0;                              \
+  virtual void SetCustomTitleForTab(Tab* tab, const std::u16string& title) {} \
```

Comment:
Should we pass `std::u16string_view` instead of `const std::u16string&`?

Reply:
Well, I'm on the fence. As the text will be gone from the `views::Textfield`, it might be safer pass u16string, TabSlotController don't have to care about lifecycle of the `title`. What do you think?